backend/src/api/candidates.js

```diff
@@ -38,10 +38,40 @@ router.post(
 )
 
 router.get(
-  '/:candidateId',
+  '/initialize-status',
   errorWrap(async (req, res) => {
-    const candidate = await Candidate.findById(req.params.candidateId)
-    res.json({ result: candidate })
+    await Candidate.update({}, { status: 'pending' }, { multi: true }, err => {
+      if (err) {
+        console.log(err.message)
+      }
+    })
```

Comment:
`errorWrap` catches all of the errors, and `middleware/errorHandler` handles it so you dont need this part